3.20.0 seems to break marshmallow_sqlalchemy

As you can see in the title, my CI just broke:
```
/env/lib/python3.11/site-packages/marshmallow_sqlalchemy/schema.py:143: in <module>
    class SQLAlchemySchema(
/env/lib/python3.11/site-packages/marshmallow/schema.py:116: in __new__
    klass._declared_fields = mcs.get_declared_fields(
E   TypeError: SQLAlchemySchemaMeta.get_declared_fields() missing 1 required positional argument: 'dict_cls'
```
Possibly an issue on their end more than yours, but someone had to report it somewhere. :-)

Same here:

```
../../../.cache/pypoetry/virtualenvs/alertmanager-N7xkbwbh-py3.10/lib/python3.10/site-packages/marshmallow_sqlalchemy/__init__.py:9: in <module>
    from .schema import (
../../../.cache/pypoetry/virtualenvs/alertmanager-N7xkbwbh-py3.10/lib/python3.10/site-packages/marshmallow_sqlalchemy/schema.py:143: in <module>
    class SQLAlchemySchema(
../../../.cache/pypoetry/virtualenvs/alertmanager-N7xkbwbh-py3.10/lib/python3.10/site-packages/marshmallow/schema.py:116: in __new__
    klass._declared_fields = mcs.get_declared_fields(
E   TypeError: SQLAlchemySchemaMeta.get_declared_fields() missing 1 required positional argument: 'dict_cls'
```

Downgrading to 3.19 solve that.

--- a/src/marshmallow/schema.py
+++ b/src/marshmallow/schema.py
@@ -117,6 +117,7 @@ class SchemaMeta(ABCMeta):
             klass=klass,
             cls_fields=cls_fields,
             inherited_fields=inherited_fields,
+            dict_cls=dict,
         )
         return klass
 

